plugins/org.python.pydev.shared_core/src/org/python/pydev/shared_core/string/StringUtils.java

```diff
@@ -1635,6 +1635,36 @@ public static boolean endsWith(final StringBuffer str, char c) {
         return false;
     }
 
+    public static boolean beginsWith(FastStringBuffer str, char c) {
```

Comment:
I couldn't find any code using this in the review (so, the addition of the `beginsWith` methods to the `StringUtils` should be reverted)... 

Also, there's a `FastStringBuffer.startsWith` which should be used if that was the case (instead of rolling a different implementation).

Reply:
That's right, I forgot to not include it in the update push... This is just part of an approach I did not follow up